rest-tests/src/test/java/io/syndesis/qe/rest/tests/publicapi/EnvironmentsPublicApiSteps.java

```diff
@@ -0,0 +1,49 @@
+package io.syndesis.qe.rest.tests.publicapi;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.syndesis.qe.endpoints.publicendpoint.EnvironmentsPublicEndpoint;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class EnvironmentsPublicApiSteps {
+
+    @Autowired
+    private EnvironmentsPublicEndpoint environmentsEndpoint;
+
+    @Then("check that number of tags is (\\w+)$")
+    public void checkNumberOfTags(int numberOfTags) {
+        assertThat(getAllTags().size()).isEqualTo(numberOfTags);
+    }
+
+    @Then("check that tag with name ([^\"]*) is in the tag list$")
+    public void checkTagExist(String tagName) {
+        List<String> list = getAllTags();
+        assertThat(list).contains(tagName);
+    }
+
+    @Then("check that tag with name ([^\"]*) is not in the tag list$")
+    public void checkTagNotExist(String tagName) {
+        List<String> list = getAllTags();
+        assertThat(list).doesNotContain(tagName);
```

Comment:
refactor these two methods to `assertThat(getAllTags())...` or even better to `assertThat(environmentsEndpoint.getAllEnvironments())...` if the `getAllTags()` is not necessary